src/Components/CreateImageWizard/steps/Oscap/index.tsx

```diff
@@ -130,80 +147,173 @@ const OscapContent = () => {
+          Compliance policy or OpenSCAP profile you want your image to be
+          compliant-ready for.{' '}
+          {lightspeedEnabled ? 'Red Hat Lightspeed' : 'Insights'} Compliance
+          allows the use of tailored policies, whereas OpenSCAP provides default
+          versions. This step automatically helps monitor the adherence of your
+          registered RHEL systems to a selected policy or profile.
+        </Content>
+        <FormGroup>
+          <Switch
+            id='fips-enabled-switch'
+            label='Enable FIPS mode'
+            isChecked={fips.enabled}
+            onChange={(_event, checked) => handleFipsToggle(checked)}
+            hasCheckIcon
           />
-          <ToggleGroupItem
-            text='OpenSCAP profiles'
-            buttonId='toggle-group-openscap'
-            isSelected={complianceType === 'openscap'}
-            onChange={() => handleTypeChange('openscap')}
+          <FormHelperText>
+            <HelperText>
+              Enable FIPS 140-2 compliant cryptographic algorithms. This setting
+              is applied at build time and persists on boot.
+            </HelperText>
+          </FormHelperText>
+        </FormGroup>
+        {environments.includes('wsl') && (
+          <Alert
+            variant='warning'
+            isInline
+            title='OpenSCAP profiles are not compatible with WSL images.'
           />
-        </ToggleGroup>
-      )}
-      {Array.isArray(policies?.data) &&
-        policies.data.length === 0 &&
-        complianceType === 'compliance' && (
-          <Alert variant='info' isInline title='No compliance policies created'>
-            <p>
-              Currently there are no compliance policies in your environment. To
-              help you get started, select one of the default policies below and
-              we will create the policy for you. However, in order to modify the
-              policy or to create a new one, you must go through{' '}
-              {lightspeedEnabled ? 'Red Hat Lightspeed' : 'Insights'}
-              Compliance.
-            </p>
-            <AlertActionLink
-              component='a'
-              onClick={() => {
-                if (!process.env.IS_ON_PREMISE) {
-                  analytics.track(
-                    `${AMPLITUDE_MODULE_NAME} - Outside link clicked`,
-                    {
-                      step_id: 'step-oscap',
-                      account_id:
-                        userData?.identity.internal?.account_id || 'Not found',
-                    },
-                  );
-                }
-              }}
-              href={COMPLIANCE_URL}
-            >
-              Save blueprint and navigate to{' '}
-              {lightspeedEnabled ? 'Red Hat Lightspeed' : 'Insights'} Compliance
-            </AlertActionLink>
-          </Alert>
         )}
-      <Oscap />
-    </Form>
+
+        <FormGroup>
+          <Content className='pf-v6-u-pb-sm'>
```

Comment:
Sorry, should have been clearer about the gating - only the compliance radio should be gated behind `complianceEnabled`, meaning OpenSCAP should be available through all the environment, but compliance should be still unavailable in prod:stable

The step in prod:stable currently looks like this:
<img width="1122" height="509" alt="image" src="https://github.com/user-attachments/assets/add30bbf-9045-4103-afc9-a8bff3b5cf14" />